src/Peachpie.Runtime/Context.Console.cs

```diff
@@ -45,6 +45,9 @@ protected void IntializeArgvArgc(params string[] args)
 
             this.Globals["argv"] = PhpValue.Create(argv);
             this.Globals["argc"] = PhpValue.Create(args.Length);
+
+            this._superglobals.server["argv"] = PhpValue.Create(argv.DeepCopy());
```

Comment:
please see the code above, keep consistency and do not use `this._superglobals.server`